### DISCUSSION TITLE
CouchDB Repository RHEL 7 broken

### DISCUSSION THREAD

**Discussion by nelli81** [apache/couchdb#4298] [General] [answered]
Hello
I'm trying to install CouchDB on a RHEL 7 System. For the installation, i'm following the procedure mentioned in https://docs.couchdb.org/en/3.2.2-docs/install/unix.html

Adding the repository (point 1.1.1.1) was possible. 

But, unfortunately, i was not able to install CouchDB (point 1.1.1.2). I always get an 404 error, the repository was not found. The following page isn't available:

https://apache.jfrog.io/artifactory/couchdb-rpm/el7Server/x86_64/repodata/repomd.xml

Could you please verify if the repository is broken?

Thanks

**Comment by nickva** [comment, marked as answer]
The repository package is are at: https://apache.jfrog.io/ui/native/couchdb-rpm/el7/x86_64/ notice the `el7` vs `el7Server`. I don't remember if we have to add a symlink for it or how that works. We've only tested with CentOS 7 as I don't have a license for RHEL 7 proper.

In the the `*.repo` file that value come automatically from https://couchdb.apache.org/repo/couchdb.repo

```init
[couchdb]
name=couchdb
baseurl=https://apache.jfrog.io/artifactory/couchdb-rpm/el$releasever/$basearch/
gpgkey=https://couchdb.apache.org/repo/keys.asc https://couchdb.apache.org/repo/rpm-package-key.asc
gpgcheck=1
repo_gpgcheck=1
enabled=1
```

So it seems we need something else there instead of `el$releasever`?

EDIT: For now I naively copied all the el7 files to el7Server and el8 to el8Server. @nelli81, give it another try perhaps?

**Comment by nelli81** [comment]
Hello

Thanks for your solution. Now, i'm able to install couchdb from the repo...

nelli81
